src/baldr/connectivity_map.cc

```diff
@@ -114,7 +114,7 @@ namespace valhalla {
 namespace baldr {
 connectivity_map_t::connectivity_map_t(const boost::property_tree::ptree& pt) {
```

Comment:
should we try to remove as many of these as possible and just pass the graph reader?